draft-peabody-dispatch-new-uuid-format-03.txt

```diff
@@ -323,84 +354,157 @@ Internet-Draft               new-uuid-format               February 2022
 
 4.  Format
 
-   The UUID length of 16 octets (128 bits) remains unchanged.  The
-   textual representation of a UUID consisting of 36 hexadecimal and
-   dash characters in the format 8-4-4-4-12 remains unchanged for human
-   readability.  In addition the position of both the Version and
-   Variant bits remain unchanged in the layout.
+   The UUID format is 16 octets; some bits of the eight octet variant
+   field specified below determine finer structure.
+
+4.1.  Variant and Version Fields
+
+   The variant bits utilized by UUIDs in this specification remain in
+   the same octet as originally defined by [RFC4122], Section 4.1.1.
+
+   For UUIDv6 the version and variant field placement from [RFC4122] are
+   unchanged and a new version (0110b [6]) has been added.
+
+   00000000-0000-6000-8000-000000000000
+   00000000-0000-6000-9000-000000000000
+   00000000-0000-6000-A000-000000000000
+   00000000-0000-6000-B000-000000000000
+   xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx
+
+               Figure 1: UUIDv6 Version and Variant Examples
+
+   M:  Version (0110b [6])
 
+   N:  Variant (1000b [8], 1001b [9], 1010b [A] or 1011b [B]) are all
+      valid values
 
+   For UUID versions 7 and 8, the variant field has been incremented and
+   the version field has been moved to the same octet as the variant.
```

Comment:
'Guess I'll make a nuisance of myself by asking that we revert this `var_ver` field concept until #26 has been resolved.

As I said, I feel it's incumbent upon @bradleypeabody to make a better case for why a new variant is needed, so I'd prefer it if we didn't just default to this in the event that conversation peters out.